src/vma/iomux/epoll_wait_call.cpp

```diff
@@ -195,6 +195,10 @@ bool epoll_wait_call::_wait(int timeout)
 		ready_fds = orig_os_api.epoll_wait(m_epfd, m_p_ready_events, m_maxevents, timeout);
 	}
 
+	if (is_suspend_required()) {
+		usleep(1);
```

Comment:
For multi threads scenario the usleep(1) is used to suspend the current thread and provide right ordering in the epfd_info::increase_ring_ref_count_no_lock() method. The issue was reproduced only for KVM.